gossipd/gossipd_wire.csv

```diff
@@ -16,7 +16,7 @@ msgdata,gossipd_init,announceable,wireaddr,num_announceable
 msgdata,gossipd_init,dev_gossip_time,?u32,
 msgdata,gossipd_init,dev_fast_gossip,bool,
 msgdata,gossipd_init,dev_fast_gossip_prune,bool,
-msgdata,gossipd_init,ip_discovery,bool,
+msgdata,gossipd_init,ip_discovery,u32,
```

Comment:
That's a huge type for 3 value, don't we have a `u8` or `u16`? :-)

Reply:
Yes I also tried smaller types before, but it raises compile errors, i.e. when using u16:
`note: expected ‘u16 *’ {aka ‘short unsigned int *’} but argument is of type ‘enum opt_autobool *’`

I guess the compiler chooses enums having 32 bits internally (even on 64bit machines).

Reply:
Well, not a big problem, these are internal messages, so the additional 2 bytes won't cost us that much :-)